src/components/ChipList.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Box, Chip, makeStyles, Tooltip } from '@material-ui/core';
+
+import { naLabel } from '../constants';
+
+const useStyles = makeStyles(theme => ({
+  chip: { margin: '3px 5px 3px 0' },
+  tooltip: {
+    backgroundColor: theme.palette.background.paper,
+    border: `1px solid ${theme.palette.grey[300]}`,
+    color: theme.palette.text.primary,
+  },
+}));
+
+function ChipList({ items }) {
+  const classes = useStyles();
+
+  if (!items || items.length === 0) return naLabel;
+
+  return items.map((item, index) => {
+    const ContainerComponent = item.tooltip
```

Comment:
This component is being redefined on every iteration of the map on every re render. To avoid this, let's defined outside of the `ChipList` component with it's own instance of `makeStyles` for its styles. It can be defined in this same file.